internal/stack/serverless.go

```diff
@@ -273,10 +288,26 @@ func (sp *serverlessProvider) BootUp(options Options) error {
 		printUserConfig(options.Printer, config)
 	}
 
-	logger.Infof("Starting local agent")
-	err = sp.startLocalAgent(options, config)
+	logger.Infof("Starting local services")
+	err = sp.startLocalServices(options, config)
 	if err != nil {
-		return fmt.Errorf("failed to start local agent: %w", err)
+		return fmt.Errorf("failed to start local services: %w", err)
+	}
+
+	if settings.LogstashEnabled {
+		//Re-create clients if running on an existing project
+		if sp.kibanaClient == nil {
+			if err = sp.createClients(project); err != nil {
+				return err
+			}
+		}
+		// Updating the output with ssl certificates created in startLocalServices
+		// This is needed because the client ssl certificates are updated for every
+		// call to stack up even though the project is already setup.
+		err = project.UpdateLogstashFleetOutput(sp.profile, sp.kibanaClient)
```

Comment:
Update would still be needed to post the certificates to fleet. But probably instead of checking for a `nil kibanaClient` we can check if the project is already created